beanstalk_worker/worker_start_test.py

```diff
@@ -208,7 +208,17 @@ def test_job_data(job_data):
     try:
         job = bs.reserve()
         job_data = json.loads(job.body)
-        test_job_data(job_data)
+        if len(sys.argv) > 1 and sys.argv[1] == "ci":
+            # If working in CI environment, don't run the scanner
+            logger.log(level=logging.INFO, msg="CI check...")
+            job_data["action"] = "start_delivery"
+            job_data["msg"] = ""
+            job_data["logs"] = ""
```

Comment:
Okay. And afaik, I have instantiated both `msg` and `logs` with `""`. It's happening at two places. lines 215-216 and 58-59